pkg/background/generate/cleanup/controller.go

```diff
@@ -71,41 +66,42 @@ func NewController(
 	npInformer kyvernov1informers.PolicyInformer,
 	urInformer kyvernov1beta1informers.UpdateRequestInformer,
 	namespaceInformer corev1informers.NamespaceInformer,
-	log logr.Logger,
-) (*Controller, error) {
-	c := Controller{
-		kyvernoClient: kyvernoclient,
+) Controller {
+	return &controller{
 		client:        client,
+		kyvernoClient: kyvernoclient,
 		pInformer:     pInformer,
 		urInformer:    urInformer,
+		pLister:       pInformer.Lister(),
+		npLister:      npInformer.Lister(),
+		urLister:      urInformer.Lister().UpdateRequests(config.KyvernoNamespace()),
```

Comment:
Is it `config.KyvernoNamespace()` or `config.KyvernoNamespace`? Or both should work?

Reply:
`config.KyvernoNamespace()` in main, `config.KyvernoNamespace` in release-1.7

Reply:
👍